adding_data_brief.qmd

```diff
@@ -23,6 +23,30 @@ schema <- get_schema()
 definitions <- austraits$definitions
 ```
 
-This chapter gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
+This section gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
 
-XXX add details
+It is important that all steps are followed so that our automated workflow proceeds without problems.
+
+1. Clone the `traits.build` repository from github
+2. Create a new branch in the repo, named for the new `dataset_id` in `author_year` format, e.g. `Gallagher_2014`.
+3. Create a new folder within the folder `data` with the name `dataset_id`, e.g. `Gallagher_2014`.
+4. Prepare the file `data.csv` and place it within the new folder ([details](#csv_file) here).
+5. Prepare the file `metadata.yml` and place it within the new folder ([details](#metadata_file) here).
+6. Add the new study into the build framework and rebuild AusTraits, by running `build_setup_pipeline()`.
+
+This step updates the file `remake.yml` with appropriate rules for the new dataset; similarly if you remove datasets, do the same. (At this stage, [remake](https://github.com/richfitz/remake) offers no looping constructs, so for now we generate the remake file using: [whisker](https://github.com/edwindj/whisker).)
+
+You can then rebuild AusTraits, including your dataset.
+
+7. Run tests and quality checks on the newly added dataset and correct the `data.csv` and `metadata.yml` files as necessary.
+8. Generate and proofread a report on the data. In particular, check that numeric trait values fall within a logical range relative to other studies, and that individual trait observations are not unnecessarily excluded because their trait values are unsupported.
+9. Return to step 6 if changes are made to the `data.csv` or `metadata.yml` files.
+10. Push the GitHub branch.
+
+The best place to get started learning how to add datasets is to work through a series of 7 [tutorials](tutorial_dataset_1.html). Each introduces you to specific [traits.build functions](https://traitecoevo.github.io/traits.build/reference/index.html) designed to facilitate the addition of dataset metadata or the metadata formats for specific types of datasets. 
+
+It may also help to download one of the [existing datasets](https://github.com/traitecoevo/austraits.build/tree/master/data) to use as a template for your own files and a guide on required content. 
```

Comment:
the template maybe easier. AusTraits is very big!

Reply:
the template just has two very simple datasets - so I've referenced both